mmv1/third_party/terraform/services/composer/resource_composer_environment.go.erb

```diff
@@ -338,39 +338,34 @@ func ResourceComposerEnvironment() *schema.Resource {
 										Optional:    true,
 										Computed:    true,
 										ForceNew:    true,
-										ConfigMode:  schema.SchemaConfigModeAttr,
```

Comment:
I'll need to double check if removing it will be a breaking check.

Reply:
No worries, I ran some of the existing tests as well when doing a bit of local development and at least confirmed from my end that the tests were still valid before creating this PR

Reply:
Removing SchemaConfigModeAttr is indeed a breaking change and can't be included in a minor release. We're currently at phase of working on 6.0.0 major release, and SchemaConfigModeAttr will be removed from all the resources. This work has already been done and 6.0.0 should be out in the next few weeks.